src/safeds/exceptions/_data.py

```diff
@@ -16,8 +16,18 @@ class UnknownColumnNameError(KeyError):
         The name of the column that was tried to be accessed.
     """
 
-    def __init__(self, column_names: list[str]):
-        super().__init__(f"Could not find column(s) '{', '.join(column_names)}'")
+    def __init__(self, column_names: list[str], similar_columns: list[str] | None = None):
+        class _UnknownColumnNameErrorMessage(
+            str,
+        ):  # This class is necessary for the newline character in a KeyError exception. See https://stackoverflow.com/a/70114007
+            def __repr__(self) -> str:
+                return str(self)
+
+        super().__init__(
+            _UnknownColumnNameErrorMessage(
+                f"Could not find column(s) '{', '.join(column_names)}'.\nDid you mean '{similar_columns}'?",
```

Comment:
If `_get_similar_columns()` did not find any similar columns, we should probably omit the `Did you mean`.

Could also leave the `similar_columns` as an optional parameter and print the old message if that parameter `None`.

Reply:
Thanks. We'll update the code with an `if similar_columns is not None: error_message += f"\nDid you mean '{similar_columns}'?"`

Reply:
Better check that
1. `similar_columns is not None`
2. `len(similar_columns` > 0
Only then add it to the error message.

Reply:
Ohh I thought this was resolved. Looking into it.

Reply:
We already have such a test case. Have you seen the second test case in `tests/safeds/data/tabular/containers/_table/test_get_similar_columns.py`? Except we'll have to move it, which i find not necessary.

Reply:
That's a test for `_get_similar_columns` directly.
I was talking abour a test case that ensures that the `UnknownColumnNameError` can be passed an empty list in the constructor.

Reply:
Please review the added file.